src/i18n.json

```diff
@@ -79,7 +79,11 @@
         "app.Upload.UploadButton": "Upload file",
         "app.Upload.Delete": "Delete",
         "app.Upload.OneFileHint": "You may only upload one file!",
-        "app.Upload.FileTooLarge": "This file is too large. Please do not upload files larger than 10 GB",
+        "app.Upload.FileTooLarge": "This file is too large.",
+        "app.Upload.FileSize": "The maximum file size is: ",
+        "app.Upload.Quota": "You have already used ",
+        "app.Upload.QuotaOf": " of ",
+        "app.Upload.QuotaUsed": "",
```

Comment:
This mustn't be empty. Please insert at least a space bar, otherwise the translation key is shown.

Reply:
Fixed